Goobi/pages/incMeta/Strukturdaten.jsp

```diff
@@ -90,11 +93,45 @@
                         <x:updateActionListener value="#{Metadaten.bildNummer}" property="#{Metadaten.pageNumber}"/>
                     </a4j:commandLink>
                 </h:panelGrid>
+                <h:panelGrid columns="2" width="100%" id="addServeralGroup" rendered="#{Metadaten.addServeralStructuralElementsMode}">
+                    <h:outputText value="#{msgs.count}: " />
+                    <h:inputText value="#{Metadaten.elementsCount}" id="elementsCount"/>
+                    <x:selectOneMenu value="#{Metadaten.addMetaDataType}">
+                        <f:selectItems value="#{Metadaten.addableMetaDataTypes}" />
+                    </x:selectOneMenu>
+                    <h:inputText value="#{Metadaten.addMetaDataValue}"/>
+                </h:panelGrid>
             </htm:td>
         </htm:tr>
         <htm:tr>
-            <htm:td styleClass="eingabeBoxen_row3" align="right">
-                <h:commandLink action="#{Metadaten.KnotenAdd}" value="#{msgs.strukturelementHinzufuegen}" target="links" />
+            <htm:td styleClass="eingabeBoxen_row3">
+                <h:commandLink action="#{Metadaten.addNodesClick}" value="#{msgs.strukturelementHinzufuegen}"
+                    target="links" style="float:right;" rendered="#{not Metadaten.addServeralStructuralElementsMode}" />
+                <htm:script rendered="#{Metadaten.addServeralStructuralElementsMode}">
+                    <h:outputText value="    function checkAddServeralStructuralElements(){
+        var inputBox = document.getElementById('formular2:elementsCount'); 
+        var count = inputBox.value;
+        if (count != '' && !isNaN(count)) {
+            var value = parseInt(count);
+            if(value > 28) {
```

Comment:
Our use case is to have years in the meta-data element. I mistyped the year in the count field and created 1982 structure elements. So there should be a warning when you are adding many elements. How much is _many_? I just guessed that 29 could be a good trade-off. I think it’s rare that books have so many chapters, and if it ever happens, well, it’s just a warning, and you can still confirm to do so.